opentreemap/treemap/templates/base.html

```diff
@@ -12,6 +12,7 @@
     <meta charset="utf-8">
     <title>{% block title %}OpenTreeMap{% block instance_title %}{% endblock %}{% block page_title %}{% endblock %}{% endblock title %}</title>
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
The MSDN [Specifying legacy document modes](https://msdn.microsoft.com/en-us/library/jj676915(v=vs.85).aspx) page makes this statement about `<meta http-equiv="X-UA-Compatible" content="IE=edge">`

> Note that this is functionally equivalent to using the HTML5 doctype. 

Seeing that we have a `<!DOCTYPE html>` at the top of our template, I removed the `X-UA-Compatible` tag and did not see any console errors or map problems when browsing the map page in IE 11 (via https://www.browserstack.com)